content_styles/remoteOperator.css

```diff
@@ -1,7 +1,7 @@
 body {
     pointer-events: none;
-    width: 100vw;
-    height: 100vh;
+    width: 100svw; /* safe view width and height work with the iPhone floating url bar (sadly vw/vh don't) */
+    height: 100svh;
```

Comment:
Great to have these and keep svw and svh in mind for future css modifications!

Reply:
Quick note for clarification, since I also didn't know about svw/svh: it stands for small view width and height and is pretty neat in general for situations where browsers might want to adjust their own UI and change the viewport as a result https://developer.mozilla.org/en-US/docs/Web/CSS/length#small

Reply:
Quick note for clarification, since I also didn't know about svw/svh: it stands for small view width and height and is pretty neat in general for situations where browsers might want to adjust their own UI and change the viewport as a result https://developer.mozilla.org/en-US/docs/Web/CSS/length#small

Reply:
Good clarification! I'll add that link as a comment for future reference